marketing_crm_partner/tests/test_lead.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Tecnativa S.L. - Jairo Llopis
+# Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2024 Ahmet Yiğit Budak (https://github.com/yibudak)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class LeadCase(TransactionCase):
```

Comment:
Use BaseCommon as base test class to reduce overhead from tracking and boost the test suite.
```suggestion
from odoo.addons.base.tests.common import BaseCommon


class LeadCase(BaseCommon):
```